dev/managedservicesplatform/stacks/monitoring/monitoring.go

```diff
@@ -640,3 +672,46 @@ func createRedisAlerts(
 
 	return nil
 }
+
+func createSentryAlerts(
+	stack cdktf.TerraformStack,
+	id resourceid.ID,
+	vars Variables,
+	channel slackconversation.Conversation,
+	slackIntegration datasentryorganizationintegration.DataSentryOrganizationIntegration,
+) error {
+	for _, config := range []sentryalert.Config{
+		{
+			SentryProject: vars.SentryProject,
+			AlertConfig: sentryalert.AlertConfig{
+				Id:        "all-issues",
+				Name:      "Notify in Slack",
+				Frequency: 15, // Notify for an issue at most once every 15 minutes
+				Conditions: []sentryalert.Condition{
+					{
+						Id:       sentryalert.EventFrequencyCondition,
+						Value:    pointers.Ptr(0), // Always (seen more than 0 times) during interval
+						Interval: pointers.Ptr("15m"),
+					},
+				},
+				ActionMatch: sentryalert.ActionMatchAny,
+				Actions: []sentryalert.Action{
+					{
+						Id: sentryalert.SlackNotifyServiceAction,
+						ActionParameters: map[string]any{
+							"workspace":  slackIntegration.Id(),
+							"channel":    channel.Name(),
+							"channel_id": channel.Id(),
+							"tags":       fmt.Sprintf("%s", vars.Service.ID),
+						},
+					},
+				},
+			},
+		},
+	} {
+		if _, err := sentryalert.New(stack, id, config); err != nil {
```

Comment:
I think it's a bit more reassuring to have the ID uniqueness guaranteed at callsite (though we might not do this consistently, maybe this is the better direction)

```suggestion
		if _, err := sentryalert.New(stack, id.Group(config.AlertConfig.Id), config); err != nil {
```

Reply:
inside `sentryalert.New` we can use less unique groupings, such as 

```
_ = issuealert.NewIssueAlert(scope, id.TerraformID("alert"), &issuealert.IssueAlertConfig{
```

Then the final ID could look like:

```
all-issues.alert
```